src/Psalm/Config.php

```diff
@@ -359,6 +366,11 @@ class Config
      */
     public $seal_all_properties = false;
 
+    /**
+     * @var bool
+     */
+    public $seal_all_properties = false;
+
```

Comment:
It's already declared few lines above

Reply:
oops, I accidentally duplicated this when I was trying to decide where to put the new property. Fixing.

Reply:
Fixed. Thanks!